tests/platform/linux/linux-xrandr-testSpec.js

```diff
@@ -0,0 +1,53 @@
+/*
+GPII Acceptance Testing
+
+Copyright 2017 Inclusive Design Research Centre, OCAD University.
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+
+The research leading to these results has received funding from the European Union's
```

Comment:
This paragraph doesn't apply anymore. You should use this one instead (https://github.com/GPII/universal/blob/master/gpii/node_modules/singleInstance/index.js#L9-L13)